### PR TITLE
Integrated Workflow of Experiments using Jupyter Notebooks: From Experimental Design to Publication

### PR DESCRIPTION
**Pull request by esimbort**

### CODE REVIEW

File: docs/mosqueda/usecase.md [hunk 1]
```diff
@@ -1,43 +1,53 @@
...
 
 ### Citation and Licensing
 
 * Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
 
-* Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
+* [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
+* [Mosqueda et al. (2017)](https://www.buffalo.edu/mceer/catalog.host.html/content/shared/www/mceer/publications/MCEER-13-0003.detail.html)
```
Comment (kks32) [inline docs/mosqueda/usecase.md]: Please fix this section and the GPL link

File: docs/mosqueda/usecase.md [hunk 2]
```diff
@@ -1,43 +1,53 @@
...
 
 ### Description 
 
-Enim ut sem viverra aliquet.  Nisi scelerisque eu ultrices vitae auctor. Scelerisque viverra mauris in aliquam.  Ut morbi tincidunt augue interdum velit euismod in pellentesque massa. Sagittis purus sit amet volutpat consequat mauris nunc congue nisi. Sed adipiscing diam donec adipiscing tristique.  In pellentesque massa placerat duis. Tortor condimentum lacinia quis vel eros donec ac. Sed velit dignissim sodales ut eu sem. Adipiscing elit duis tristique sollicitudin nibh sit amet commodo. Quis risus sed vulputate odio ut.
+The main objective of the present Jupiter Notebook is to develop an interactive computing platform able to perform system identification procedures of a three degree of freedom system based on real test results, as well as offer all the potential users a tool to start learning system identification on a simple structure.
 
 [Link Example - this goes to Google](https://www.google.com)
```
Comment (kks32) [inline docs/mosqueda/usecase.md]:
```suggestion
```

File: docs/mosqueda/usecase.md [hunk 3]
```diff
@@ -1,43 +1,53 @@
...
 
 * This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
+* [Mosqueda et al. (2017)](https://www.buffalo.edu/mceer/catalog.host.html/content/shared/www/mceer/publications/MCEER-13-0003.detail.html)
 
 ### Description 
 
-Enim ut sem viverra aliquet.  Nisi scelerisque eu ultrices vitae auctor. Scelerisque viverra mauris in aliquam.  Ut morbi tincidunt augue interdum velit euismod in pellentesque massa. Sagittis purus sit amet volutpat consequat mauris nunc congue nisi. Sed adipiscing diam donec adipiscing tristique.  In pellentesque massa placerat duis. Tortor condimentum lacinia quis vel eros donec ac. Sed velit dignissim sodales ut eu sem. Adipiscing elit duis tristique sollicitudin nibh sit amet commodo. Quis risus sed vulputate odio ut.
+The main objective of the present Jupiter Notebook is to develop an interactive computing platform able to perform system identification procedures of a three degree of freedom system based on real test results, as well as offer all the potential users a tool to start learning system identification on a simple structure.
```
Comment (kks32) [inline docs/mosqueda/usecase.md]:
```suggestion
The main objective of the present Jupyter Notebook is to develop an interactive computing platform able to perform system identification procedures of a three degree of freedom system based on real test results, as well as offer all the potential users a tool to start learning system identification on a simple structure.
```

File: docs/mosqueda/usecase.md [hunk 4]
```diff
@@ -1,43 +1,53 @@
...
 
 [Link Example - this goes to Google](https://www.google.com)
 
-## Header 2
+## Analysis Cases
+
+The system identification of the structure is carried out for three analysis cases:
+1.	White noise signal
```
Comment (kks32) [inline docs/mosqueda/usecase.md]:
```suggestion

1.	White noise signal
```

File: docs/mosqueda/usecase.md [hunk 5]
```diff
@@ -1,43 +1,53 @@
...
+The abstract of the Jupiter notebook describes the experimental program designed to capture the structural response of base isolated building when it impacts against a surrounding moat wall under extreme ground motions.
 
-In aliquam sem fringilla ut morbi. Interdum varius sit amet mattis vulputate enim nulla aliquet. Sit amet mattis vulputate enim nulla.  In egestas erat imperdiet sed euismod nisi porta lorem. Eget nulla facilisi etiam dignissim diam.  Facilisi cras fermentum odio eu feugiat. Velit aliquet sagittis id consectetur. Vel quam elementum pulvinar etiam.  Ut diam quam nulla porttitor massa id neque aliquam. Sodales ut etiam sit amet nisl.  Scelerisque varius morbi enim nunc faucibus a. Sit amet volutpat consequat mauris nunc. Et leo duis ut diam.
 
 *Add images to the folder img and use relative path to specify the location of the image.*
```
Comment (kks32) [inline docs/mosqueda/usecase.md]:
```suggestion
```

File: docs/mosqueda/usecase.md [hunk 6]
```diff
@@ -1,43 +1,53 @@
...
+## About Technical Report MCEER-13-0003
 
+The results that are used in the system identification analysis are obtained from a test program consisting of shake table testing of a quarter-scale model representative of the three-story IMRF at the Structural Engineering and Earthquake Simulation Laboratory (SEESL) at the University at Buffalo.
+The base isolated IMRF model was selected for the experimental program due to availability of scaled members and simplicity in fabrication. However, both base isolated buildings including the IMRF and OCBF are used in numerical simulations presented latter in this report. The principal objective of these experiments was to generate experimental data for base isolated building pounding a moat wall, specifically the impact force and the superstructure response. This data is subsequently used to validate the numerical models of structural pounding. In addition to generating unique data on structural pounding, the behavior of a base isolated building impacting different types of moat wall installed at different gap distance was also of investigated. [extract from Technical Report MCEER-13-0003.]
 Numbered list
```
Comment (kks32) [inline docs/mosqueda/usecase.md]:
```suggestion
```
Remove these

**Review by kks32 — CHANGES_REQUESTED** [review]
Please move code comments to the Jupyter code located in Community Data. Remove repeated code here and only use this doc to explain the overall structure and not a line-by-line summary. 

Please add Jupyter to Community Data and fix all spelling of Jupyter.

Please add some relevant figures/plots.

File: docs/mosqueda/usecase.md (hunk 1 above)
Comment (esimbort) [reply docs/mosqueda/usecase.md]: The references have been updated.

File: docs/mosqueda/usecase.md (hunk 2 above)
Comment (esimbort) [reply docs/mosqueda/usecase.md]: The document was improved.

File: docs/mosqueda/usecase.md (hunk 6 above)
Comment (esimbort) [reply docs/mosqueda/usecase.md]: The document has been improved.

File: docs/mosqueda/usecase.md (hunk 5 above)
Comment (esimbort) [reply docs/mosqueda/usecase.md]: All the images have been added to the folder img.

File: docs/mosqueda/usecase.md (hunk 3 above)
Comment (esimbort) [reply docs/mosqueda/usecase.md]: The spelling errors have been removed.

File: docs/mosqueda/usecase.md (hunk 4 above)
Comment (esimbort) [reply docs/mosqueda/usecase.md]: The document has been modified.

**Comment by esimbort** [comment]
A complete draft has been submitted. However, we still need to replace the link to Jupyter Notebooks once they are in community data.

File: docs/mosqueda/usecase.md [hunk 7]
```diff
@@ -1,89 +1,112 @@
...
-* Please cite [AUTHORS et al. (20xx) - example of published project](https://doi.org/10.17603/ds2-3zdj-493) to acknowledge the use of any resources from this use case.
-
-* Please cite [Rathje et al. (2017)](https://doi.org/10.1061/(ASCE)NH.1527-6996.0000246) to acknowledge the use of DesignSafe resources.  
-
-* This software is distributed under the GNU General Public License (https://www.gnu.org/licenses/gpl-3.0.html).  
-
-### Description 
-
-Enim ut sem viverra aliquet.  Nisi scelerisque eu ultrices vitae auctor. Scelerisque viverra mauris in aliquam.  Ut morbi tincidunt augue interdum velit euismod in pellentesque massa. Sagittis purus sit amet volutpat consequat mauris nunc congue nisi. Sed adipiscing diam donec adipiscing tristique.  In pellentesque massa placerat duis. Tortor condimentum lacinia quis vel eros donec ac. Sed velit dignissim sodales ut eu sem. Adipiscing elit duis tristique sollicitudin nibh sit amet commodo. Quis risus sed vulputate odio ut.
+1. [Rathje et al. (2020)](https://doi.org/10.3389/fbuil.2020.547706) “Enhancing research in natural hazard engineering through the DesignSafe cyberinfrastructure”. Frontiers in Built Environment, 6:213.
```
Comment (kks32) [inline docs/mosqueda/usecase.md]: Please note that this is not a reference list. Here only show how to cite your paper relevant to this use case, Rathje 2018 for DesignSafe and the Software license GPL

**Review by kks32 — CHANGES_REQUESTED** [review]
Please update your citation and license section

**Review by kks32 — APPROVED** [review]

**Review by kks32 — APPROVED** [review]

**Pull request merged**